rakelib/test.rake

```diff
@@ -87,6 +87,28 @@ namespace "test" do
     task.reenable
   end
 
+  task "integration" => ["setup"] do
```

Comment:
I wonder if we should add here something like for others test, that they do the bootstrap automatically and you only need to run ```rake test:integration```, what do you think?